purchase_requisition_auto_rfq/tests/test_purchase_requisition.py

```diff
@@ -0,0 +1,80 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import tests
+
+
+class TestPurchaseRequisition(tests.SavepointCase):
+    def setUp(self):
+        super().setUp()
+        self._init_products()
+        self._init_requisitions()
+
+    def _init_products(self):
+        self._kitchenset = self.env.ref('purchase_requisition_auto_rfq.kitchenset')
+        self._blankets = self.env.ref('purchase_requisition_auto_rfq.blankets')
+        self._tarpaulin = self.env.ref('purchase_requisition_auto_rfq.tarpaulin')
+
+    def _init_requisitions(self):
+        uom = self._kitchenset.uom_po_id
+        self._requisition1 = self.env['purchase.requisition'].create({
+            'name': "PR01",
+            'line_ids': [
+                (0, False, {'product_id': self._kitchenset.id, 'product_qty': 10, 'product_uom_id': uom.id}),
```

Comment:
You should change this code for fix flake8.
```suggestion
                (0, False, {'product_id': self._kitchenset.id,
                            'product_qty': 10, 'product_uom_id': uom.id}),
```